pkg/features/pilot/pilot.go

```diff
@@ -58,4 +67,20 @@ var (
 
 	// AzDebug indicates whether to log service registry az info.
 	AzDebug = os.Getenv("VERBOSE_AZ_DEBUG") == "1"
+
+	// NetworkScopes isolates namespaces, limiting configuration for
+	// egress and other mesh services to only hosts defined in same namespace or
+	// 'admin' namespaces. Using services from any other namespaces will require the new NetworkScope
+	// config. In most cases 'istio-system' should be included.
+	NetworkScopes = os.Getenv("DEFAULT_NAMESPACE_DEPENDENCIES")
```

Comment:
Have more comments like  format "ns1,ns2,ns3"

Reply:
Ok ( this is a short-lived env - will be replaced by values.yaml before release - but should have comments)